packages/lodestar/src/db/api/beacon/repositories/blockArchive.ts

```diff
@@ -45,7 +45,7 @@ export class BlockArchiveRepository extends BulkRepository<BeaconBlock> {
       .map((datum) => deserialize(this.type, datum))
```

Comment:
Not exactly related to this PR, but it seems strange that `safeLowerLimit` is set to a number or Buffer.

Reply:
Yeah, I’ve been thinking the same thing. I assumed it has some significance, so I never decided to delete it in case it broke something somewhere else.

Reply:
It seems to work as expected, even with `Buffer.alloc(0)`. It should probably be just `0` though.
We can change in a later PR